examples/sagemaker/endpoint.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: Endpoint
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpoint
+  labels:
+    testing.upbound.io/example-name: e
+  name: e
+spec:
+  forProvider:
+    endpointConfigNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: ec
+    region: us-east-1
+    tags:
+      Name: foo
+---
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: EndpointConfiguration
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpointconfiguration
+    upjet.upbound.io/manual-intervention: "It requires an image which can be create/deleted only using AWS CLI"
+  labels:
+    testing.upbound.io/example-name: ec
+  name: ec
+spec:
+  forProvider:
+    productionVariants:
+    - initialInstanceCount: 1
+      instanceType: ml.t2.medium
+      modelNameSelector:
+        matchLabels:
+          testing.upbound.io/example-name: example
+      variantName: variant-1
+    region: us-east-1
+    tags:
+      Name: foo
+---
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: Model
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpointconfiguration
+    upjet.upbound.io/manual-intervention: "It requires an image which can be create/deleted only using AWS CLI"
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    executionRoleArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    primaryContainer:
+      - image: ${data.aws_account_id}.dkr.ecr.us-east-1.amazonaws.com/sagemaker-scikit-learn:latest
```

Comment:
Is this the only reason for the manual intervention annotation? I'm not very familiar with Sagemaker, but couldn't you just define the endpoint and EndpointConfiguration in here, and not have a model?

Reply:
You need to have a model for an EndpointConfiguration and need an EndpointConfiguration for an Endpoint hence the need for the label